### ISSUE TITLE
Add arc-theme in regolith-look

### ISSUE THREAD

**Issue by akss13** [regolith-linux/regolith-desktop#267]
https://github.com/horst3180/arc-theme

**Comment by kgilmer** [comment]
https://regolith-linux.org/docs/developer/looks/

**Comment by JoshuaCrewe** [comment]
I’d be interested in this too. Has anyone made a start on it? I’ll get to reading up on how it would be done. If anyone has a starting point already we can work from that!

**Comment by JoshuaCrewe** [comment]
There is already a bit of a leg up in the sense that these theme configs probably already exist on the internet. Arc is itself a GTK theme and there is already an oficial [Rofi theme](https://github.com/davatorium/rofi-themes/tree/master/Official%20Themes). Whats missing? The i3 bar?

In the nord `root` file there are references to other config files :
```
! -- Applications
! These files map values defined above into specific app settings.
#include "/etc/regolith/styles/st-term"
#include "/etc/regolith/styles/i3-wm"
#include "/etc/regolith/styles/i3xrocks"
#include "/etc/regolith/styles/rofi"
#include "/etc/regolith/styles/gnome"
```
If I am understanding that comment correctly we fill out `color`, `typeface` and `theme` and we get styles for those apps for free?

**Comment by kgilmer** [comment]
@JoshuaCrewe yes that's correct.  The [readme here](https://github.com/regolith-linux/regolith-styles) gives more detail as to how the various styles files relate.  Good luck and we're always happy to answer questions, either here or in slack.

**Comment by JoshuaCrewe** [comment]
Brilliant! I made a start on this and have a repo I need to publish. I’ll do that and link it if anyone wants to join in. But for the moment work has taken over all my mental budget :)

**Comment by JoshuaCrewe** [comment]
@kgilmer I have started trying to map the colours from the arc theme to a regolith look template and it is a bit trial and error. Which is fine. I can make it better by spending less time in a VM and get a proper dev environment setup.

My question is that there already exists Arc themes for various bits and bobs. The rofi one I already incorporated. Should I be including the Arc GTK theme in somehow? That would make the Files app look the part right? (sorry I am a bit new to this side of things). Is there any guidance on what the variables in colors is used for, specifically base1-3 and base01-03 ?

**Comment by kgilmer** [comment]
Hi @JoshuaCrewe !

>  Should I be including the Arc GTK theme in somehow? 

Yes, you would specify the GTK theme with the `gtk_theme` define, which would then get loaded here: https://github.com/regolith-linux/regolith-styles/blob/master/Xresources/gnome#L1-L2

> That would make the Files app look the part right?

I believe so, yes.

> Is there any guidance on what the variables in colors is used for, specifically base1-3 and base01-03 ?

The existing color names are based on the original theme: Solarized.  You can see how they are referring to bases here: https://ethanschoonover.com/solarized/

Sorry, there isn't a clean schema for the color designations.  We have considered migrating to base16 but it's a lot of work, so things remain as they are for the time being.

**Comment by lbibass** [comment]
Is there an update to this? Would love to have this theme in Regolith.

**Comment by kgilmer** [comment]
The blocker at this point is producing a valid debian package for the Ark GTK theme.  Due to some symlink build stuff, proper packaging has been difficult.  There are a few open source GTK themes with clean, sane builds, such as the pop-os GTK theme, but I haven't had time yet to look into using for an Arc gtk theme.
